src/mlnet/Program.cs

```diff
@@ -23,7 +23,7 @@ class Program
         public static void Main(string[] args)
         {
             var telemetry = new MlTelemetry();
-
+            int exitCode = 0;
```

Comment:
Might recommend defaulting to 1, as there's many more unknown ways to fail then paths to success.

Reply:
_(unintentionally bad advice for life)_

Reply:
```suggestion
            int exitCode = 1;
```